zx.mjs

```diff
@@ -101,10 +101,12 @@ async function writeAndImport(script, filepath, origin = filepath) {
 
 async function importPath(filepath, origin = filepath) {
   let ext = extname(filepath)
+  let tmpFilename= Math.random().toString(36).substr(2) + '.mjs'
```

Comment:
One thing to consider: error messages.

User will get errors with file name is random which can lead to bad DX. 

What if `filename + '.mjs'` is not exists? We can use it (same as old behavior).

In case there is a collision use: `filename + random + '.mjs'`.

WDYF?

Reply:
Oh, good call. Yeah, that makes sense to me. Will push that change later today!

Reply:
Hi @antonmedv, sorry for the delay. 

Just pushed the requested fix. If you feel like it's good to go, I'll let you hit resolve conversation. If you'd like me to make any additional fixes, though, just let me know. Thanks!